lib/stimpack/integrations/rspec.rb

```diff
@@ -1,19 +1,46 @@
 module Stimpack
   module Integrations
     class RSpec
-      def self.install(app)
-        return unless defined?(::RSpec)
-        # Sometimes, the `rspec-rails` gem is installed in the development
-        # group. This means the ::RSpec module will be defined. However, this
-        # doesn't mean that we've loaded rspec-core, or even about to run tests.
-        # Let's make sure we are actually loading the test environment before
-        # installing our integration. An easy of doing this is seeing of the
-        # configuration method exists on RSpec.
-        return unless ::RSpec.respond_to?(:configuration)
+      def initialize
+        # This is the list of directories RSpec was told to run.
+        to_run = ::RSpec.configuration.instance_variable_get(:@files_or_directories_to_run)
```

Comment:
This kind of looks like private API of rspec -- are we comfortable relying on it?

Reply:
It is, but it hasn't changed for quite some time. I think I can find a better way or work with RSpec folks to get this right. In the meantime, we'll rely on this and we can ensure we can continue to support this.

Reply:
Sounds good. If they stop supporting it hopefully our tests break pretty dramatically 🤞